app/views/spree/admin/slides/_form.html.erb

```diff
@@ -61,10 +61,10 @@
     <div class='col-2'>
       <%= f.field_container :image do %>
         <% if f.object.image? %>
-          <p><%= image_tag f.object.image, style: 'max-width: 100%; height: auto;' %></p>
+          <p><%= image_tag f.object.image.url, style: 'max-width: 100%; height: auto;' %></p>
```

Comment:
What about moving the style into a class? Not a blocker but it is only a suggestion 👀 💦

Reply:
oh that will be pretty good